sass/main.scss

```diff
@@ -118,7 +116,6 @@ figure {
 }
 
 code {
-  font-family: Hack, DejaVu Sans Mono, Monaco, Consolas, Ubuntu Mono, monospace;
```

Comment:
Moved the font family to the font file.

Reply:
Hmmm, I don't see this `font-family` anywhere now. Are you sure it won't break things?

Reply:
It's defined [here](https://github.com/pawroman/zola-theme-terminimal/blob/a5b96f006ffbdf460f91c5a73f1b3b6864d32b24/sass/main.scss#L16). I've tested the changes locally but it doesn't hurt if you give it a try to sanity check.